main branch: test failures for  TestWALSegmentSizeBounds, TestMaxBlockChunkSegmentSizeBounds

using main (9486be535bbfbff65d858f94720b5e467caead4d) , running `make`, some of the tests fail. is this normal?
```
$ make
(...)
>> building binaries
GO111MODULE=on /home/dieter/go/bin/promu build --prefix /home/dieter/go/src/github.com/prometheus/prometheus 
 >   prometheus
 >   promtool
>> running all tests
GO111MODULE=on go test -race  ./...
--- FAIL: TestWALSegmentSizeBounds (1.49s)
    main_test.go:212: ts=2021-10-13T09:50:46.314Z caller=level.go:63 level=info msg="No time or size retention was set so using the default time retention" duration=15d
        ts=2021-10-13T09:50:46.315Z caller=level.go:63 level=info msg="Starting Prometheus" version="(version=, branch=, revision=)"
        ts=2021-10-13T09:50:46.315Z caller=level.go:63 level=info build_context="(go=go1.17.1, user=, date=)"
        ts=2021-10-13T09:50:46.315Z caller=level.go:63 level=info host_details="(Linux 5.14.11-arch1-1 #1 SMP PREEMPT Sun, 10 Oct 2021 00:48:26 +0000 x86_64 dieter-m6800 (none))"
        ts=2021-10-13T09:50:46.315Z caller=level.go:63 level=info fd_limits="(soft=500000, hard=500000)"
        ts=2021-10-13T09:50:46.315Z caller=level.go:63 level=info vm_limits="(soft=unlimited, hard=unlimited)"
        ts=2021-10-13T09:50:46.373Z caller=level.go:63 level=info component=web msg="Start listening for connections" address=0.0.0.0:9090
        ts=2021-10-13T09:50:46.374Z caller=level.go:63 level=error msg="Unable to start web listener" err="listen tcp 0.0.0.0:9090: bind: address already in use"
        
    main_test.go:223: prometheus should be still running: exit status 1
    main_test.go:212: ts=2021-10-13T09:50:46.629Z caller=level.go:63 level=info msg="No time or size retention was set so using the default time retention" duration=15d
        ts=2021-10-13T09:50:46.629Z caller=level.go:63 level=info msg="Starting Prometheus" version="(version=, branch=, revision=)"
        ts=2021-10-13T09:50:46.629Z caller=level.go:63 level=info build_context="(go=go1.17.1, user=, date=)"
        ts=2021-10-13T09:50:46.629Z caller=level.go:63 level=info host_details="(Linux 5.14.11-arch1-1 #1 SMP PREEMPT Sun, 10 Oct 2021 00:48:26 +0000 x86_64 dieter-m6800 (none))"
        ts=2021-10-13T09:50:46.629Z caller=level.go:63 level=info fd_limits="(soft=500000, hard=500000)"
        ts=2021-10-13T09:50:46.629Z caller=level.go:63 level=info vm_limits="(soft=unlimited, hard=unlimited)"
        ts=2021-10-13T09:50:46.655Z caller=level.go:63 level=info component=web msg="Start listening for connections" address=0.0.0.0:9090
        ts=2021-10-13T09:50:46.655Z caller=level.go:63 level=error msg="Unable to start web listener" err="listen tcp 0.0.0.0:9090: bind: address already in use"
        
    main_test.go:212: ts=2021-10-13T09:50:46.924Z caller=level.go:63 level=info msg="No time or size retention was set so using the default time retention" duration=15d
        ts=2021-10-13T09:50:46.925Z caller=level.go:63 level=info msg="Starting Prometheus" version="(version=, branch=, revision=)"
        ts=2021-10-13T09:50:46.925Z caller=level.go:63 level=info build_context="(go=go1.17.1, user=, date=)"
        ts=2021-10-13T09:50:46.925Z caller=level.go:63 level=info host_details="(Linux 5.14.11-arch1-1 #1 SMP PREEMPT Sun, 10 Oct 2021 00:48:26 +0000 x86_64 dieter-m6800 (none))"
        ts=2021-10-13T09:50:46.925Z caller=level.go:63 level=info fd_limits="(soft=500000, hard=500000)"
        ts=2021-10-13T09:50:46.925Z caller=level.go:63 level=info vm_limits="(soft=unlimited, hard=unlimited)"
        ts=2021-10-13T09:50:46.957Z caller=level.go:63 level=info component=web msg="Start listening for connections" address=0.0.0.0:9090
        ts=2021-10-13T09:50:46.957Z caller=level.go:63 level=error msg="Unable to start web listener" err="listen tcp 0.0.0.0:9090: bind: address already in use"
        
    main_test.go:212: Error parsing commandline arguments: units: unknown unit  in 10
        
    main_test.go:212: ts=2021-10-13T09:50:47.452Z caller=level.go:63 level=info msg="No time or size retention was set so using the default time retention" duration=15d
        ts=2021-10-13T09:50:47.453Z caller=level.go:63 level=info msg="Starting Prometheus" version="(version=, branch=, revision=)"
        ts=2021-10-13T09:50:47.453Z caller=level.go:63 level=info build_context="(go=go1.17.1, user=, date=)"
        ts=2021-10-13T09:50:47.453Z caller=level.go:63 level=info host_details="(Linux 5.14.11-arch1-1 #1 SMP PREEMPT Sun, 10 Oct 2021 00:48:26 +0000 x86_64 dieter-m6800 (none))"
        ts=2021-10-13T09:50:47.453Z caller=level.go:63 level=info fd_limits="(soft=500000, hard=500000)"
        ts=2021-10-13T09:50:47.453Z caller=level.go:63 level=info vm_limits="(soft=unlimited, hard=unlimited)"
        ts=2021-10-13T09:50:47.487Z caller=level.go:63 level=info component=web msg="Start listening for connections" address=0.0.0.0:9090
        ts=2021-10-13T09:50:47.487Z caller=level.go:63 level=error msg="Unable to start web listener" err="listen tcp 0.0.0.0:9090: bind: address already in use"
        
--- FAIL: TestMaxBlockChunkSegmentSizeBounds (0.55s)
    main_test.go:254: ts=2021-10-13T09:50:47.714Z caller=level.go:63 level=info msg="No time or size retention was set so using the default time retention" duration=15d
        ts=2021-10-13T09:50:47.714Z caller=level.go:63 level=info msg="Starting Prometheus" version="(version=, branch=, revision=)"
        ts=2021-10-13T09:50:47.715Z caller=level.go:63 level=info build_context="(go=go1.17.1, user=, date=)"
        ts=2021-10-13T09:50:47.715Z caller=level.go:63 level=info host_details="(Linux 5.14.11-arch1-1 #1 SMP PREEMPT Sun, 10 Oct 2021 00:48:26 +0000 x86_64 dieter-m6800 (none))"
        ts=2021-10-13T09:50:47.715Z caller=level.go:63 level=info fd_limits="(soft=500000, hard=500000)"
        ts=2021-10-13T09:50:47.715Z caller=level.go:63 level=info vm_limits="(soft=unlimited, hard=unlimited)"
        ts=2021-10-13T09:50:47.749Z caller=level.go:63 level=info component=web msg="Start listening for connections" address=0.0.0.0:9090
        ts=2021-10-13T09:50:47.749Z caller=level.go:63 level=error msg="Unable to start web listener" err="listen tcp 0.0.0.0:9090: bind: address already in use"
        
    main_test.go:254: ts=2021-10-13T09:50:48.005Z caller=level.go:63 level=info msg="No time or size retention was set so using the default time retention" duration=15d
        ts=2021-10-13T09:50:48.005Z caller=level.go:63 level=info msg="Starting Prometheus" version="(version=, branch=, revision=)"
        ts=2021-10-13T09:50:48.005Z caller=level.go:63 level=info build_context="(go=go1.17.1, user=, date=)"
        ts=2021-10-13T09:50:48.005Z caller=level.go:63 level=info host_details="(Linux 5.14.11-arch1-1 #1 SMP PREEMPT Sun, 10 Oct 2021 00:48:26 +0000 x86_64 dieter-m6800 (none))"
        ts=2021-10-13T09:50:48.005Z caller=level.go:63 level=info fd_limits="(soft=500000, hard=500000)"
        ts=2021-10-13T09:50:48.005Z caller=level.go:63 level=info vm_limits="(soft=unlimited, hard=unlimited)"
        ts=2021-10-13T09:50:48.038Z caller=level.go:63 level=info component=web msg="Start listening for connections" address=0.0.0.0:9090
        ts=2021-10-13T09:50:48.039Z caller=level.go:63 level=error msg="Unable to start web listener" err="listen tcp 0.0.0.0:9090: bind: address already in use"
        
    main_test.go:265: prometheus should be still running: exit status 1
FAIL
```

```
uname -srm
Linux 5.14.11-arch1-1 x86_64
```

--- a/cmd/prometheus/main_test.go
+++ b/cmd/prometheus/main_test.go
@@ -202,7 +202,7 @@ func TestWALSegmentSizeBounds(t *testing.T) {
 	}
 
 	for size, expectedExitStatus := range map[string]int{"9MB": 1, "257MB": 1, "10": 2, "1GB": 1, "12MB": 0} {
-		prom := exec.Command(promPath, "-test.main", "--storage.tsdb.wal-segment-size="+size, "--config.file="+promConfig)
+		prom := exec.Command(promPath, "-test.main", "--storage.tsdb.wal-segment-size="+size, "--web.listen-address=0.0.0.0:0", "--config.file="+promConfig)
 
 		// Log stderr in case of failure.
 		stderr, err := prom.StderrPipe()
@@ -244,7 +244,7 @@ func TestMaxBlockChunkSegmentSizeBounds(t *testing.T) {
 	}
 
 	for size, expectedExitStatus := range map[string]int{"512KB": 1, "1MB": 0} {
-		prom := exec.Command(promPath, "-test.main", "--storage.tsdb.max-block-chunk-segment-size="+size, "--config.file="+promConfig)
+		prom := exec.Command(promPath, "-test.main", "--storage.tsdb.max-block-chunk-segment-size="+size, "--web.listen-address=0.0.0.0:0", "--config.file="+promConfig)
 
 		// Log stderr in case of failure.
 		stderr, err := prom.StderrPipe()

